.github/workflows/build-test.yml

```diff
@@ -573,15 +488,6 @@ jobs:
           beakerx_kernel_java install
           conda install pytorch cpuonly -c pytorch
 
-      # Install PyTorch with pip
-      - name: Install PyTorch on Windows and Linux
-        if: matrix.python != 'conda' && matrix.python != 'noPython' && (matrix.os == 'windows-latest' || matrix.os == 'ubuntu-latest')
-        run: pip install torch==1.7.1+cpu torchvision==0.8.2+cpu torchaudio===0.7.2 -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
None of these are used,
We have these for some old tests that relate to fancy tooltips for tensor varaibles (and that functionality is not in the extension at all).